index.js

```diff
@@ -4,27 +4,23 @@ module.exports = function (img, selection_canvas, onCrop) {
   if('function' === typeof selection_canvas)
     onCrop = selection_canvas, selection_canvas = null
 
-  var c2 = selection_canvas = selection_canvas || h('canvas.hypercrop__selection', {width: 512, height: 512})
-
   onCrop = onCrop || function () {}
 
- var width = img.width, height = img.height
+  var c2 = selection_canvas = selection_canvas || h('canvas.hypercrop__selection', {width: 512, height: 512})
```

Comment:
good question! I've only been testing this in Patchbay, where I don't think there is a canvas passed in (haven't checked closely).

I think that could be an issue worth noting on the repo. I don't think my extensions are touching the existing logic here, so would like to make that a seperate PR.